src/utils/utils.ts

```diff
@@ -367,6 +365,14 @@ export const getTokenDataByAddresses = async (
   psp22: PSP22,
   address: string
 ): Promise<Record<string, Token>> => {
+  const knownTokens: Record<string, Token> = {}
```

Comment:
i don't think we should prevent getting known tokens data inside this function and we should prevent passing known tokens instead in other parts of the app

Reply:
for this one, i had the problem that two tokens from URL were loaded as unknown tokens, but they were I mainnet token list.